doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3311 @@
+{
```

Comment:
<p>This section is about logp inference, but the content is about rng seeding</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
Removed the seeding part and just left a small pymc model and show where PyTensor appears. Is this ok?